snapshots/urbanization/2024-12-02/ghsl_urban_centers.py

```diff
@@ -0,0 +1,25 @@
+"""Script to create a snapshot of dataset. This version of the dataset was provided directly by the source via email."""
```

Comment:
Might be worth to leave the e-mail of the contact here (or in the .dvc), so we can reach out to them directly when updating. But up to you!